### ISSUE TITLE
How to turn colored output off in nbdiff?

### ISSUE THREAD

**Issue by stas00** [jupyter/nbdime#426]
I'm trying to configure nbdime to be used with [git-notifier](https://github.com/rsmmr/git-notifier), which  emails the commit diffs as plain text. 

Would it be possible to add an option not to generated colored output as most (all?) mail clients can't render this in text mode and so I get garbled text?

My configuration is the default:

```
$ git config --get diff.jupyternotebook.command
git-nbdiffdriver diff

```

Thank you.

**Comment by vidartf** [comment]
Thanks for the feedback! I found [some appropriate TODOs](https://github.com/jupyter/nbdime/blob/c968e6faa706e99e24c4c5f0f13052d9665c6950/nbdime/prettyprint.py#L33-L36) in the code that I'm currently working on locally.

**Comment by vidartf** [comment]
Released in 1.0.3
